tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_HalfSpaceMirror.cpp

```diff
@@ -77,20 +78,20 @@ SPECTRE_TEST_CASE(
   pypp::SetupLocalPythonEnvironment local_python_env{"PointwiseFunctions"};
   const Elasticity::ConstitutiveRelations::IsotropicHomogeneous<dim>
       constitutive_relation{36.36363636363637, 30.76923076923077};
-  const HalfSpaceMirrorProxy solution{0.177, constitutive_relation, 350};
+  const HalfSpaceMirrorProxy solution{0.177, constitutive_relation, 350, 1e-10};
```

Comment:
Also set the relative tolerance explicitly :) Perhaps set both to `1e-12` so that they are below the test-tolerance of `1e-10` that you set below.

Reply:
I am afraid that `integration_intervals` will blow up, but will try.